module-2/data-analysis-with-pandas/your-code/Untitled.ipynb

```diff
@@ -0,0 +1,1282 @@
+{
```

Comment:
- The first thing you should  is to have a clear overview of the characteristics of your data sets (total number of records, type of data (categorical vs numerical, time series vs cross-section vs data-panel data, ....), number of missing values and distribution among columns, etc....)
- One of your best friends will be info () method and also a library called pandas-profiling. You should install pip install pandas-profiling on your term. You then would import pandas profiling in your notebook and run the following method: 
```Python 
pandas_profiling.ProfileReport(df)
```
- Good approach!! It's a good practice to include in your notebook the meaning of each variable in your data set. 
- Your notebook does not have "a logic thread". What do you want to do with your data set? What are the questions you need to be answered with your analysis? Your code throw results but this isn't data analysis.